cg/models/cg_config.py

```diff
@@ -360,11 +353,14 @@ class RunInstruments(BaseModel):
     illumina: IlluminaConfig
 
 
+class RunDirectoryNamesServices(BaseModel):
+    pacbio: PacbioRunDirectoryNamesService
+    model_config = ConfigDict(arbitrary_types_allowed=True)
```

Comment:
Why do we need `arbitrary_types_allowed`?

Reply:
It only happens when the types are inherited types, same happened with the PostProcessingService

Reply:
Could you elaborate? What happens and which object are you referring to as inherited?

Reply:
As far as I know this has nothing to do with inheritance. By default your own custom classes are not allowed as a field in a BaseModel - they should be standard types or other BaseModels (might be some caveats): https://docs.pydantic.dev/latest/api/config/#pydantic.config.ConfigDict.arbitrary_types_allowed

Reply:
But this does not happen with the RunInstruments class in line 350

Reply:
That's because they inherit from BaseModel

Reply:
So the attributes of the `RunInstruments` class are pydantic objects that inherit directly from BaseModel, while the atreibutes of `RunDirectoryNamesServices` are pydantic objects that inherit directly from `RunNamesService`. That is where the issue is I think

Reply:
As far as I can tell, the attributes of RunDirectoryNamesServices are not pydantic objects at all

Reply:
Ohh yes my bad, they inherit from ABC

Reply:
Can I resolve this? @Vince-janv @islean

Reply:
Yes